docs/CSL-1925.md

```diff
@@ -0,0 +1,256 @@
+
+# Problem Definition
+
+An address in Cardano is always derived from a root key using BIP32 style derivation
+except it uses a different elliptic curve than Bitcoin (Ed25519). 
+
+The problem initially raised concerns exchanges which have only one account per root
+key and thus 2^31 possible addresses. In this scenario according to the birthday
+paradox there is a non negligible chance of a collision occuring.
+
+The first child (or branch) of the root key is derived according to the _account index_
+which by default is 2^31 or in binary `10000000000000000000000000000000` where the
+first bit represents the hardened state. As a BIP32 path this can be represented as 
+`m/aH`. The second child is derived according to an _address index_.
+
+Both the account and address indices fall within the range 2^31 in terms of how many
+addresses can be derived.
+
+In the current scheme the address size is 76 bytes, where each index takes up 5 bytes.
```

Comment:
If we want to be picky, we could mention that is 4 bytes for the `Word32` storage plus 1 byte for CBOR serialisation overhead. Not essential by any means though 😀

Reply:
I modified to include these finer details. :)